login as: root
root@89.111.132.126's password:
Welcome to Ubuntu 18.04 LTS (GNU/Linux 4.15.0-112-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Tue Sep 21 12:11:35 MSK 2021

  System load:  0.36               Processes:           145
  Usage of /:   22.0% of 58.09GB   Users logged in:     0
  Memory usage: 44%                IP address for ens3: 89.111.132.126
  Swap usage:   0%

 * Super-optimized for small spaces - read how we shrank the memory
   footprint of MicroK8s to make it the smallest full K8s around.

   https://ubuntu.com/blog/microk8s-memory-optimisation

  Get cloud support with Ubuntu Advantage Cloud Guest:
    http://www.ubuntu.com/business/services/cloud

 * Canonical Livepatch is available for installation.
   - Reduce system reboots and improve kernel security. Activate at:
     https://ubuntu.com/livepatch

Может быть обновлён 181 пакет.
4 обновления касаются безопасности системы.

New release '20.04.3 LTS' available.
Run 'do-release-upgrade' to upgrade to it.


*** Требуется перезагрузка системы ***
Last login: Mon Sep 20 19:07:29 2021 from 176.120.62.250
root@vm2002143548:~# sudo pm2 list
┌─────┬───────┬─────────────┬─────────┬─────────┬──────────┬────────┬──────┬───────────┬──────────┬──────────┬──────────┬──────────                                                                                                          ┐
│ id  │ name  │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching                                                                                                           │
├─────┼───────┼─────────────┼─────────┼─────────┼──────────┼────────┼──────┼───────────┼──────────┼──────────┼──────────┼──────────                                                                                                          ┤
│ 0   │ ep    │ default     │ 0.0.0   │ fork    │ 29079    │ 4M     │ 4    │ online    │ 0.6%     │ 119.8mb  │ root     │ disabled                                                                                                           │
│ 1   │ ms    │ default     │ 1.0.0   │ fork    │ 13203    │ 16h    │ 13   │ online    │ 0.8%     │ 95.2mb   │ root     │ disabled                                                                                                           │
└─────┴───────┴─────────────┴─────────┴─────────┴──────────┴────────┴──────┴───────────┴──────────┴──────────┴──────────┴──────────                                                                                                          ┘
root@vm2002143548:~# sudo pm2 list
┌─────┬───────┬─────────────┬─────────┬─────────┬──────────┬────────┬──────┬───────────┬──────────┬──────────┬──────────┬──────────┐
│ id  │ name  │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
├─────┼───────┼─────────────┼─────────┼─────────┼──────────┼────────┼──────┼───────────┼──────────┼──────────┼──────────┼──────────┤
│ 0   │ ep    │ default     │ 0.0.0   │ fork    │ 29079    │ 4M     │ 4    │ online    │ 0.7%     │ 119.8mb  │ root     │ disabled │
│ 1   │ ms    │ default     │ 1.0.0   │ fork    │ 13203    │ 16h    │ 13   │ online    │ 1.3%     │ 94.7mb   │ root     │ disabled │
└─────┴───────┴─────────────┴─────────┴─────────┴──────────┴────────┴──────┴───────────┴──────────┴──────────┴──────────┴──────────┘
root@vm2002143548:~# cd /var/www/ms
root@vm2002143548:/var/www/ms# sudo pm2 stop ms
[PM2] Applying action stopProcessId on app [ms](ids: 1)
[PM2] [ms](1) ✓
┌─────┬───────┬─────────────┬─────────┬─────────┬──────────┬────────┬──────┬───────────┬──────────┬──────────┬──────────┬──────────┐
│ id  │ name  │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
├─────┼───────┼─────────────┼─────────┼─────────┼──────────┼────────┼──────┼───────────┼──────────┼──────────┼──────────┼──────────┤
│ 0   │ ep    │ default     │ 0.0.0   │ fork    │ 29079    │ 4M     │ 4    │ online    │ 0%       │ 119.8mb  │ root     │ disabled │
│ 1   │ ms    │ default     │ 1.0.0   │ fork    │ 0        │ 0      │ 13   │ stopped   │ 0%       │ 0b       │ root     │ disabled │
└─────┴───────┴─────────────┴─────────┴─────────┴──────────┴────────┴──────┴───────────┴──────────┴──────────┴──────────┴──────────┘
root@vm2002143548:/var/www/ms# sudo node index.js
h
Mongoose: addresses.createIndex({ location: '2dsphere' }, { background: true })
Mongoose: executors.createIndex({ location: '2dsphere' }, { background: true })
Mongoose: executoroffers.createIndex({ 'address.location': '2dsphere' }, { background: true })
Mongoose: orders.createIndex({ 'address.location': '2dsphere' }, { background: true })
CONNECTED FROM index.js
Mongoose: configs.find({}, { projection: {} })
CONNECTED FROM amqp.js
Worker is started
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:27422) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 6)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 7)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 8)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 9)
(node:27422) UnhandledPromiseRejectionWarning: CastError: Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
    at new CastError (/var/www/ms/node_modules/mongoose/lib/error/cast.js:29:11)
    at ObjectId.cast (/var/www/ms/node_modules/mongoose/lib/schema/objectid.js:246:11)
    at ObjectId.SchemaType.applySetters (/var/www/ms/node_modules/mongoose/lib/schematype.js:969:12)
    at ObjectId.SchemaType._castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1383:15)
    at ObjectId.SchemaType.castForQuery (/var/www/ms/node_modules/mongoose/lib/schematype.js:1373:15)
    at ObjectId.SchemaType.castForQueryWrapper (/var/www/ms/node_modules/mongoose/lib/schematype.js:1352:15)
    at cast (/var/www/ms/node_modules/mongoose/lib/cast.js:315:32)
    at model.Query.Query.cast (/var/www/ms/node_modules/mongoose/lib/query.js:4641:12)
    at model.Query.Query._castConditions (/var/www/ms/node_modules/mongoose/lib/query.js:1843:10)
    at model.Query.<anonymous> (/var/www/ms/node_modules/mongoose/lib/query.js:2710:8)
    at model.Query._wrappedThunk [as _deleteOne] (/var/www/ms/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
    at process.nextTick (/var/www/ms/node_modules/kareem/index.js:369:33)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
(node:27422) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 10)
"idCard"






 Cast to ObjectId failed for value "idCard" at path "_id" for model "Cards"
